fix(docs): updated the quickstart for kubeflow pipelines. Fixes 4617 in (kubeflow/pipelines) (#2512)
kubeflow/website: content/en/docs/pipelines/pipelines-quickstart.md

## Deploy Kubeflow and open the pipelines UI

There are several options to [deploy Kubeflow pipelines](/docs/pipelines/installation/), follow the option that best suits your needs. If you are uncertain and just want to try out kubeflow pipelines it is recommended to start with the [standalone deployment](/docs/pipelines/installation/standalone-deployment).

> Once you have deployed kubeflow pipelines make sure you can access the UI, this step depends on the method of deployment. 

Reviewer: ```suggestion
Once you have deployed kubeflow pipelines make sure you can access the UI, how to do it varied depending on the deployment method. 
```

    alt="Run results on the pipelines UI"
    class="mt-3 mb-3 border border-info rounded">

> You can find [the source code for the data passing in python components sample](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)

Reviewer: ```suggestion
You can find [the source code for the python components](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
```
